mkdocs.yml

```diff
@@ -55,9 +55,12 @@ nav:
       - 'RSS Processing': 'platform/tutorials/rss-tutorial/rss-processing-pipeline.md'
       - 'Currency Alerting': 'platform/tutorials/currency-alerting/currency-alerting.md'
       - 'Telemetry data': 'platform/tutorials/telemetry-data/telemetry-data.md'
-      - 'Train and deploy an ML model':
-        - 'How to train an ML model': 'platform/tutorials/train-and-deploy-ml/train-ml-model.md'
-        - 'Run ML model in realtime environment': 'platform/tutorials/train-and-deploy-ml/deploy-ml.md'
+      - 'Real-time Machine Learning (ML) pipelines':
+        - platform/tutorials/train-and-deploy-ml/index.md
+        - 'Create your data': 'platform/tutorials/train-and-deploy-ml/create-data.md'
+        - 'Import data into Jupyter': 'platform/tutorials/train-and-deploy-ml/import-data.md'
+        - 'Train your ML model': 'platform/tutorials/train-and-deploy-ml/train-ml.md'
+        - 'Deploy your ML model': 'platform/tutorials/train-and-deploy-ml/deploy-ml.md'
       - 'Data Science': 'platform/tutorials/data-science/data-science.md'
```

Comment:
link has changed

Reply:
index.md i think

Reply:
You mean the link for the data science tutorial?